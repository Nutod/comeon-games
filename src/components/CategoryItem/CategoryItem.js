import React, { memo } from "react";
import PropTypes from "prop-types";

function CategoryItem({ name }) {
  return (
    <div className="category item">
      <div className="content">
        <div className="header">
          <h4>{name && name}</h4>
        </div>
      </div>
    </div>
  );
}

CategoryItem.propTypes = {
  name: PropTypes.string.isRequired
};

export default memo(CategoryItem);
