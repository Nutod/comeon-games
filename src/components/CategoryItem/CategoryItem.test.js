import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import CategoryItem from "./CategoryItem";

describe("<CategoryItem /> Snapshot", () => {
  const wrapper = shallow(<CategoryItem />);

  it("Should match snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
