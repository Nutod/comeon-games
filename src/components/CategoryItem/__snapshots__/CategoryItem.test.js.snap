// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CategoryItem /> Snapshot Should match snapshot 1`] = `
<div
  className="category item"
>
  <div
    className="content"
  >
    <div
      className="header"
    >
      <h4 />
    </div>
  </div>
</div>
`;
